guide/src/web-getting-started.md

```diff
@@ -43,35 +43,57 @@ You can import the components just like other things you’re used to in JavaScr
 import { FerrostarMap, BrowserLocationProvider } from "@stadiamaps/ferrostar-components";
 ```
 
-## Configure the `<ferrostar-map>` component
+## Configure the <ferrostar-map> Component
 
-Now you can use Ferrostar in your HTML like this:
+Now you can use Ferrostar in your web component like this:
 
-```html
+## HTML
+
+```
 <ferrostar-map
-  id="ferrostar"
+id="ferrostar"
   valhallaEndpointUrl="https://api.stadiamaps.com/route/v1"
   styleUrl="https://tiles.stadiamaps.com/styles/outdoors.json"
   profile="bicycle"
 ></ferrostar-map>
 ```
 
-Here we have used Stadia Maps URLs, which should work without authentication for local development.
-(Refer to the [authentication docs](https://docs.stadiamaps.com/authentication/)
-for network deployment details; you can start with a free account.)
+## React
```

Comment:
I believe that the first example should remain generic, and we should have separate sections for framework-specific (React, Vue, etc.) code. This appears to replace the generic code, which is applicable to a large number of users, with a React-specific example.